cub/test/test_grid_barrier.cu

```diff
@@ -47,7 +47,9 @@ using namespace cub;
 /**
  * Kernel that iterates through the specified number of software global barriers
  */
-__global__ void Kernel(GridBarrier global_barrier, int iterations)
+_CCCL_SUPPRESS_DEPRECATED_PUSH
+__global__ void Kernel(GridBarrier global_barrier, int iterations) //
+  _CCCL_SUPPRESS_DEPRECATED_POP
```

Comment:
Thinking about this a bit more, I believe the issue is that this is between the function header and the body.

Should we move it after the function?

Reply:
This does not work unfortunately:
```
_CCCL_SUPPRESS_DEPRECATED_PUSH
__global__ void Kernel(GridBarrier global_barrier, int iterations)
{
 ...
}
_CCCL_SUPPRESS_DEPRECATED_POP
```
The warning still appears in some kind of cudafe stub file.